test/server/cards/03_TWI/events/PrivateManufacturing.spec.ts

```diff
@@ -0,0 +1,46 @@
+describe('Private Manufacturing', function () {
+    integration(function (contextRef) {
+        describe('Private Manufacturing\'s ability', function () {
+            it('should draw two cards and not ask to move two cards to bottom of the deck', function () {
```

Comment:
nit: test name should include the condition so it's clearer what's going on. i.e., should end with "because there are no token units". Similar note for the other test name

Reply:
yeah, thanks for that. I had a hard time coming up with a test sentence that wasn't weird.